pom.xml

```diff
@@ -7,6 +7,45 @@
 
   <build>
     <plugins>
+
+      <plugin>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <environmentVariables>
+            <CACHE_POOL_SIZE>10</CACHE_POOL_SIZE>
+            <CACHE_SERVER>localhost:1234</CACHE_SERVER>
+            <COOKIE_DOMAIN>.companieshouse.gov.uk</COOKIE_DOMAIN>
+            <COOKIE_NAME>abc</COOKIE_NAME>
+            <COOKIE_SECRET>iamasecret</COOKIE_SECRET>
+            <DEFAULT_SESSION_EXPIRATION>3600</DEFAULT_SESSION_EXPIRATION>
+            <HUMAN_LOG>1</HUMAN_LOG>
+
+            <cdn.url>//</cdn.url>
+            <chs.url>//</chs.url>
+
+            <account.url>//</account.url>
+
+            <createAccount.url>${home.url}/create-account</createAccount.url>
+            <developerGuidelines.url>${home.url}/developer-guidelines</developerGuidelines.url>
+            <gettingStarted.url>${home.url}/getting-started</gettingStarted.url>
+            <manageApplications.url>${home.url}/manage-applications</manageApplications.url>
+            <home.url>${home.url}</home.url>
+            <piwik.siteId>//</piwik.siteId>
+            <piwik.url>//</piwik.url>
```

Comment:
Why have we put the environment variable in the pom.xml when we have the applications.properties file?

Reply:
The values for surefire only apply in unit tests.

Reply:
👍